### ISSUE TITLE
All the term of training losses are 0.000 except mel loss

### ISSUE THREAD

**Issue by jirenhui** [yl4579/StyleTTS2#159]
Great work! However, When I reused the LJSpeech to train the model, I encountered some problems as shown in the figure below. train_list, val_list and OOD_text are all original in the project. Can you help me solve it?
<img width="1043" alt="image" src="https://github.com/yl4579/StyleTTS2/assets/70502653/134c5ecb-3d0e-4c94-938c-6a759cd2127a">

**Comment by martinambrus** [comment]
That's normal for the first 49 epochs on 1st training stage. Once you get to epoch 50, you'll see the other losses as well.
